composer.json

```diff
@@ -14,7 +14,7 @@
         "phpunit/phpunit": "^4.8.36 || ^5.7.27 || ^6.5.14 || ^7.5.20 || ^8.0 || ^9.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.16"
+        "friendsofphp/php-cs-fixer": "^2"
```

Comment:
why it's needed?

Reply:
Because of this: https://github.com/sanmai/polyfill/runs/1406796543?check_suite_focus=true

It's easier than to follow a full `dev-tools`/`.phar` approach, right at the moment.

Reply:
issue is that if you run php-cs-fixer 2.0 it may apply some other CodeStyle and expect them to be applied, while we are following ruleset of 2.16.

Reply:
True, but we ain't going to run php-cs-fixer on PHP 5.6, are we?

There's a dedicated CS workflow, that uses PHP 7.4 to verify the work.